packages/regl-worldview/src/commands/Command.js

```diff
@@ -84,9 +84,14 @@ export default class Command<T> extends React.Component<Props<T>> {
 
   _getHitmapPropFromHitmapId(objectId: number) {
     const { hitmapProps = [] } = this.props;
+
     return hitmapProps.find((hitmapProp) => {
-      // TODO handle objects with 'colors' property
-      if (hitmapProp.color) {
+      if (hitmapProp.points) {
+        // for instanced rendering, find the hitmapProp that produces the same id range and return it
+        if (objectId >= hitmapProp.id && objectId < hitmapProp.id + hitmapProp.points.length) {
```

Comment:
hmm this relies on the implementation of `getHitmapId` to be `(marker, idx) => marker.id + idx`. I wonder if it makes more sense to default that implementation if this is how we go about the bool check?

Reply:
Good question! Addressed this by introducing `testObjectIdInRange` prop :)